sewer/client.py

```diff
@@ -250,18 +250,6 @@ def _request(self, method, url, **kwargs):
 
         return response
 
-    @staticmethod
```

Comment:
maybe these changes to log_response could be in their own PR?  I feel like this and the auth changes are 2 separate concerns/discussions

Reply:
#169 is on the fast path to commit.  Great appreciation if you could sanity check it, since it's All Your Fault.  :-)